src/bare-metal/alloc.md

```diff
@@ -0,0 +1,16 @@
+# `alloc`
+
+To use `alloc` you must implement a global (heap) allocator.
```

Comment:
I was thinking that the `LockedHeap` type would implement a trait, and it turns out it's called `GlobalAlloc`:

```suggestion
To use `alloc` you must implement a [global (heap) allocator](https://doc.rust-lang.org/nightly/core/alloc/trait.GlobalAlloc.html).
```

Reply:
Done.